docs/Changelog.md

```diff
@@ -14022,6 +13927,130 @@ This version of the operator has been available since version 12 of the default
 <dd>Constrain input and output types to all numerical tensor types.</dd>
 </dl>
 
+### <a name="GatherND-12"></a>**GatherND-12**</a>
+
+  Given `data` tensor of rank `r` >= 1, `indices` tensor of rank `q` >= 1, and `batch_dims` integer `b`, this operator gathers 
+  slices of `data` into an output tensor of rank `q + r - indices_shape[-1] - 1 - b`.
+  
+  `indices` is an q-dimensional integer tensor, best thought of as a `(q-1)`-dimensional tensor of index-tuples into `data`, 
+  where each element defines a slice of `data`
+  
+  `batch_dims` (denoted as `b`) is an integer indicating the number of bacth dimensions, i.e the first `b` number of dimensions of 
+  `data` tensor and `indices` are representing the batches, and the gather starts from the `b`+1 dimension. 
```

Comment:
nit: `b+1` (not `b` + 1), the difference in text highlighted in the documentation might be misleading